doc/nrf/includes/sample_board_rows.txt

```diff
@@ -30,7 +30,7 @@
 
 .. nrf9160dk_nrf52840
 
-| :ref:`nRF9160 DK <ug_nrf9160>` | PCA10090 | :ref:`nrf9160dk <nrf9160dk_nrf9160>` | ``nrf9160dk/nrf52840`` |
+| :ref:`nRF9160 DK <ug_nrf9160>` | PCA10090 | :ref:`nrf9160dk <nrf9160dk_nrf52840>` | ``nrf9160dk/nrf52840`` |
```

Comment:
Shouldn't this also have zephyr: ?

Reply:
nRF9160 is not labeled with zephyr:board:: in Zephyr - https://github.com/nrfconnect/sdk-zephyr/blob/main/boards/nordic/nrf9160dk/doc/index.rst?plain=1. Hence, we use ref in this case.

Reply:
Well, then again we have it with zephyr in applications/serial_lte_modem/doc/nRF91_as_Zephyr_modem.rst

Reply:
Yeah, I had fixed it : https://github.com/nrfconnect/sdk-nrf/pull/22348/files#diff-e8657136d907eef0ab1e3780aed43c07e82f797acbeb786f14d20a5242b653a2R18.